data_steward/validation/ehr_union.py

```diff
@@ -877,6 +888,11 @@ def create_load_aou_death(bq_client, project_id, input_dataset_id,
         `CalculatePrimaryDeathRecord` updates the table at the end of the
         Unioned EHR data tier creation.
     """
+    bq_utils.create_standard_table(AOU_DEATH,
+                                   f'{UNIONED_EHR}_{AOU_DEATH}',
+                                   drop_existing=True,
+                                   dataset_id=output_dataset_id)
+
```

Comment:
bq_utils.py is in the process of being deprecated, so `create_standard_table(...)` function will be deprecated eventually. I left a suggestion to see what you think. 

```suggestion
table_name = f'{bq_client.project_id}.{output_dataset_id}.{UNIONED_EHR}_{AOU_DEATH}'
schema_list = bq_client.get_table_schema(AOU_DEATH)
table_obj = bq.Table(table_name, schema=schema_list)
table_obj.clustering_fields = 'person_id'
table_obj.time_partitioning = bq.table.TimePartitioning(type_='DAY')
bq_client.create_table(table_obj, exists_ok=True)
```